qutebrowser/completion/models/histcategory.py

```diff
@@ -76,7 +76,7 @@ def set_pattern(self, pattern):
         # given the search term "a b", the WHERE clause would be:
         # ((url || title) LIKE '%a%') AND ((url || title) LIKE '%b%')
```

Comment:
Like this

```suggestion
        # ((url || ' ' || title) LIKE '%a%') AND ((url || ' ' || title) LIKE '%b%')
```

Reply:
Good catch! I was away for a while, I'll fix this up tonight